extra/3-magic-8-ball.js

```diff
@@ -45,8 +45,46 @@
 
 // This should log "The ball has shaken!"
 // and return the answer.
+
+const answers = {
+  "very positive" : [
+  "It is certain.",
+  "It is decidedly so.",
+  "Without a doubt.",
+  "Yes - definitely.",
+  "You may rely on it."
+  ],
+
+  "positive" : [
+  "As I see it, yes.",
+  "Most likely.",
+  "Outlook good.",
+  "Yes.",
+  "Signs point to yes."
+  ],
+
+  "negative" : [
+  "Reply hazy, try again.",
+  "Ask again later.",
+  "Better not tell you now.",
+  "Cannot predict now.",
+  "Concentrate and ask again."
+  ],
+
+  "very negative" : [
+  "Don't count on it.",
+  "My reply is no.",
+  "My sources say no.",
+  "Outlook not so good.",
+  "Very doubtful."
+  ]
+};
+
 function shakeBall() {
-  //Write your code in here
+  console.log("The ball has shaken!");
+  const allArraysAnswers = answers["very positive"].concat(answers["positive"], answers["negative"], answers["very negative"]);
```

Comment:
this was a good try 
but ideally, you should not do like that 
the object key like "very positive" might change the next day. 
what if you receive that object from some server? 
i'd need to change you code constantly.
so you should make an abstraction from data values into data structure 
and walk through object and get all answers programmatically without hardcoding those object keys